modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -7,6 +7,16 @@
 
 You must build a container disk with a virtual machine image and push it to a container registry before it can used with a virtual machine. You can then either import the container disk into a PVC using a data volume and attach it to a virtual machine, or you can attach the container disk directly to a virtual machine as an ephemeral `containerDisk` volume.
 
+[NOTE]
+====
+The size of a disk image inside a container disk is limited by the maximum layer size of the registry where the container disk is hosted. For Red Hat Quay, the default limit is 20Gi. You can configure the `MAXIMUM_LAYER_SIZE` field in the Red Hat Quay configuration schema. For more details, see the schema for Red Hat Quay configuration in the link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Manage Red Hat Quay] documentation.
+====
+
+[IMPORTANT]
```

Comment:
Both of these notes seem important. Do you think that one large `[IMPORTANT]` admonition with two bullet points would work?

Reply:
I could go either way. My only thought was that the possibility for severe performance impact should be called out as more critical. But if the two notes are that closely related, i have no problem combining them into one [Important].

cc: @fabiand for your opinion.

Reply:
My personal preference would be to keep them in two separate boxes, mainly because they are not related. If they are in one box, with wo bullets, I could quickly ignore the second bulet if I see that the first does not apply to me - in other words: I'd assume the two are related. This is just my personal note-

In the end I'd align to patterns which are already present in the docs.

Reply:
If they're totally unrelated, I can see the value of separating them. Maybe you could put the text of the first note directly into the body so that the `[IMPORTANT]` admonition stands out even more. (I hear that people sometimes just ignore `[NOTE]`s.)

Reply:
Yeah, I like that last suggestion @ousleyp. Removed the NOTE markup and integrated it into the body text. Might as well give as much light to the more important admonition.